web/packages/api/src/toEthereum_v2.ts

```diff
@@ -1198,3 +1341,99 @@ async function buildMessageId(
     ])
     return blake2AsHex(entropy)
 }
+
+function createPNASourceParachainTx(
```

Comment:
We need to make sure this case is disabled. We do not support it for initial release.

Reply:
I'd prefer to leave it here. The first released PNAs only covers the native asset on AH plus KSM from AH. So they won't touch this branch.

Reply:
I am not saying we should remove the code path. I am saying we should make sure users never hit this code path until we officially support it. Are we sure that the only assets that we list as transferable in mainnet do not trigger this code path?

Reply:
> Are we sure that the only assets that we list as transferable in mainnet do not trigger this code path?

Yeah, it's only for multi-hop transfers from a source parachain to Ethereum.